astro/src/content/quickstarts/quickstart-php-web.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: PHP
+description: Quickstart integration of PHP web application with FusionAuth
+navcategory: getting-started
+prerequisites: Docker version 20
+section: web
+cta: EmailListCTA
+technology: PHP
+language: PHP
+icon: /img/icons/php.svg
+faIcon: fa-php
+url: https://github.com/FusionAuth/fusionauth-quickstart-php-web
+codeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-php-web/main
+---
+
+import Aside from '/src/components/Aside.astro';
+import LoginAfter from '../../diagrams/quickstarts/login-after.astro';
+import LoginBefore from '../../diagrams/quickstarts/login-before.astro';
+import {RemoteCode, RemoteValue} from '@fusionauth/astro-components';
+
+
+In this quickstart, you will build an application with {frontmatter.technology} and integrate it with FusionAuth. The application is for [ChangeBank](https://www.youtube.com/watch?v=pkH-kD73QUM), a global leader in converting dollars to coins. The app will have areas reserved for logged in users and public-facing sections.
+
+<Aside type="note">
+  When developing PHP web applications, it's advisable to consider using a framework. Frameworks provide a structured environment, streamlining development and enhancing code organization. With built-in security features, community support, and time-saving tools, frameworks offer a robust foundation for efficient and maintainable projects.
+</Aside>
+
+Find the Docker Compose file and source code for the complete application at <a href={frontmatter.url}>{frontmatter.url}</a>.
+
+## Prerequisites
+
+For this quickstart, you'll need:
+
+* [PHP](https://www.php.net/downloads.php) 8+
+* [Composer](https://getcomposer.org/)
+* [Docker](https://www.docker.com) version 20 or later, which is the quickest way to start FusionAuth. (There are [other ways](/docs/v1/tech/installation-guide/).)
+
+You will run PHP 8, Composer, and Apache in Docker. Using Docker makes this tutorial suitable for Windows, Linux, and macOS.
```

Comment:
Please don't use docker for PHP. As mentioned in another quickstart (the laravel web one, I believe) we assume the developer has things installed locally. Please rewrite to not use docker for the PHP application.

Reply:
@vcampitelli but you use docker for everything :) .

Can you explain why you think this is the best approach and worth breaking consistency across all the other quickstarts (excluding the wordpress and drupal quickstarts which have a lot more going on than this one)?

Reply:
I'd say that PHP is not included in most Linux distros (at least not up-to-date versions), but you had me on "worth breaking consistency across all the other quickstarts". I have no arguments strong enough against that. No-Docker approach then!

Reply:
Thanks @vcampitelli .

@rideam can you please rework to use a locally installed PHP binary? Feel free to add that as a prerequisite and composer (or maybe you already have that?).

Reply:
@mooreds  I opened a PR here https://github.com/FusionAuth/fusionauth-quickstart-php-web/pull/1 to change the app to not use the Docker image and use locally installed PHP

Reply:
Resolving since I merged that.